src/ServiceControlInstaller.Engine/Instances/ServiceControlBaseService.cs

```diff
@@ -66,7 +66,10 @@ public string AclUrl
             }
         }
 
-        public string AclMaintenanceUrl
+        //This is an old aclurl registration for embedded RavenDB instance that includes the hostname.
+        //We need that to make sure we can clean-up old registration when removing instances created
+        //using pre 4.17 versions of ServiceControl.
+        public string LegacyAclMaintenanceUrl
```

Comment:
I think it might be better to inline this instead of having a compute only property. The comment could be moved into the RemoveUrlAcl method